src/modules/claim/claims.controller.e2e.spec.ts

```diff
@@ -200,14 +204,14 @@ describe('ClaimsController', () => {
         expect(res.body.length).toEqual(1);
         expect(res.body[0]).toBeInstanceOf(Object);
         expect(res.body[0].issuedToken).toEqual(token);
-        expect(res.body[0].issuedAt).toHaveLength(10);
+        expect(res.body[0].issuedAt).not.toBeNaN();
```

Comment:
This seems like it makes the test less specific. Could we keep both conditions? Or is the `NaN` condition more appropriate?

Reply:
I think it is more specific. `issuedAt` should be a timestamp, so nor we are checking if this is a number, but previously we were just checking for any string with length 10.

Reply:
Hmm, I think `NaN` is a type in javascript, right? So this test is saying that `issuedAt` just has to be not `NaN`. But most things are not `NaN`, e.g. a `number`, a `string`, an `object`, an `array`, etc. So I'm not seeing how this is more specific. Could you explain a bit more please?

Reply:
I am checking if `issuedAt` is not a not  number, so actually I can pass whatever type i want to NaN and checking if I could parse it to number later on. Since we have double negation we are checking if it can be parse to number